lib/compat/wordpress-6.5/fonts/fonts.php

```diff
@@ -141,8 +128,11 @@ function wp_unregister_font_collection( $collection_id ) {
 	}
 }
 
-// TODO: update to production font collection URL.
-wp_register_font_collection_from_json( 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/01aa57731575bd13f9db8d86ab80a2d74e28a1ac/releases/gutenberg-17.6/collections/google-fonts-with-preview.json' );
+function gutenberg_register_font_collections() {
+	// TODO: update to production font collection URL.
+	wp_register_font_collection( 'google-fonts', 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/main/releases/gutenberg-17.7/collections/google-fonts-with-preview.json' );
+}
+add_action( 'init', 'gutenberg_register_font_collections' );
 
```

Comment:
This is breaking the unregistering of the font collections. Using this hook makes the unregistering of the font collections fail.

Reply:
I think best practice here would be to unregister font collections using the init hook with a later priority. Otherwise a plugin that's loaded earlier would not have access to font collections that are yet to be registered by plugins that will be loaded later.

Reply:
Could we trigger the from another action, prior to init? 

When would we expect plugins to register collections (and another to be able to also unregister those collections, not just the default collection?)

Reply:
:+1: I don't like to rely on hook names to do that because that forces the extenders to understand the hooks and their order, which can be confusing. But due to there being other similar examples in core core functions, I think it's ok, and we should replicate that for consistency.